query/Cargo.toml

```diff
@@ -106,6 +106,8 @@ tonic = "=0.6.2"
 typetag = "0.1.8"
 uuid = { version = "0.8.2", features = ["serde", "v4"] }
 walkdir = "2.3.2"
+serfig = {path="../../../xuanwo/serfig"}
+serde-bridge = {path="../../../xuanwo/serde-bridge"}
```

Comment:
Can not find the file 🤔 ?

Reply:
Yep, this PR is a demo to show my idea and is not workable.

I expect to make it works tomorrow :rocket:

Reply:
The PR is buildable now!